From 88927b3293fe9829cc7a7b00ba1e5524a4337f45 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 8 Oct 2015 18:44:27 +0500
Subject: [PATCH] initramfs-live-install: drop all automount rules while
 installing

Different auto mount rules reside under /etc/udev/rules.d and
can cause trouble while installing on devices like mmc as
these get auto mounted and the installer script fails
to format them.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 init-install.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init-install.sh b/init-install.sh
index aa17b5f..c9a3534 100644
--- init-install.sh
+++ init-install.sh
@@ -111,6 +111,7 @@ device=$TARGET_DEVICE_NAME
 #
 rm -f /etc/udev/rules.d/automount.rules
 rm -f /etc/udev/scripts/mount*
+rm -f /etc/udev/rules.d/*auto-mount.rules
 
 #
 # Unmount anything the automounter had mounted
-- 
1.9.1

